f5/bigip/tm/cm/trust.py

```diff
@@ -31,13 +31,17 @@ class Add_To_Trust(UnnamedResourceMixin, ExclusiveAttributesMixin,
 
     def __init__(self, cm):
         super(Add_To_Trust, self).__init__(cm)
+        base_uri = type(self).__name__.replace('_', '-').lower()
```

Comment:
We should never manually manipulate classes-into-uris that should always be done in a single place:

https://github.com/F5Networks/f5-common-python/blob/development/f5/bigip/resource.py#L165

To be clear, we should always and only use this --^ method.